lib/jdl/jdl_importer.js

```diff
@@ -102,6 +103,18 @@ function makeJDLImporter(content, configuration) {
   };
 }
 
+function parseConfiguration(configuration) {
```

Comment:
it doesn't exactly do what its name implies, I'd have named it "setupCreationTimestampFromConfiguration"

Reply:
What about "setupConfiguration" to keep it generic.

Reply:
Yes, let's